build/Signing/SignToolTemplateConfig.json

```diff
@@ -5,9 +5,6 @@
             "strongName": null,
             "values": [
                 "Microsoft.VisualStudio.ProjectSystem.CSharp.Templates.vsix",
```

Comment:
This VSIX is no longer created.  `Microsoft.VisualStudio.Templates.CSharp.NetCore.vsmanproj` is the only thing that is still built and its only purpose is to package `Microsoft.NET.Sdk.swixproj`.